CONTENTSTYLEGUIDE.md

```diff
@@ -78,25 +76,24 @@ The first one directs the user to the baseline document for manual checking. The
 ```
 # At this time we are unable to test for X because of Y
 tests[{
-    "Requirement" : "The matching policy bulletpoint from the baseline document goes here",
-    "Control" : "Example 2.1",
+    "PolicyId" : PolicyId,
     "Criticality" : "Shall/Not-Implemented",
-    "Commandlet" : "",
+    "Commandlet" : [],
     "ActualValue" : [],
-    "ReportDetails" : "Currently cannot be checked automatically. See (Product) Secure Configuration Baseline policy 2.# for instructions on manual check",
+    "ReportDetails" : NotCheckedDetails(PolicyId),
```

Comment:
I missed updating the style guide completely (thank you!).  May be worth adding that 
import data.report.utils.NotCheckedDetails
must be included.

Reply:
I agree adding a section over imports is important, but that is more syntax vs style. The style guide are best practices for readability. Maybe we should pick up making that Rego cheat sheet again?